lightning/src/ln/features.rs

```diff
@@ -665,6 +672,24 @@ impl<T: sealed::Context> Features<T> {
 			(byte & unknown_features) != 0
 		})
 	}
+
+	// Returns true if the features within `self` are a subset of the features within `other`.
+	pub(crate) fn is_subset(&self, other: &Self) -> bool {
+		for (idx, byte) in self.flags.iter().enumerate() {
```

Comment:
I wonder if there's a slightly simpler way to check, bitwise, if `(self & other) == self`. But if there's not, the explicit iteration is extremely legible.